backend/app/services/resource_library_service.py

```
@@ -461,25 +464,83 @@ def list_published(
+            .limit(limit)
+            .all()
+        )
+        index_page_ms = (perf_counter() - phase_started_at) * 1000
+
+        phase_started_at = perf_counter()
+        kind_ids = [publication.kind_id for publication in publications]
+        resources = (
+            db.query(Kind)
+            .filter(
+                Kind.id.in_(kind_ids),
+                Kind.is_active == True,
```

Ruff (0.16.0)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)